lineage/LineageInputOutput.py

```diff
@@ -203,3 +203,30 @@ def tryRecursion(pColumn, lower, upper, parentCell, currentLineage, lineageSizeI
     # add daughter to current Lineage
     currentLineage.append(daughterCell)
     return daughterCell
+
+def fullRecursive(cell, a):
+    """ To plot the <full> binary tree based on inter-mitotic time (lifetime) of cells.
+    a should be: a = [Clade(lineage1.full_lineage[0].obs[2]+lineage1.full_lineage[0].obs[3])]
+    """
+    if cell.isLeaf():
+        return Clade(branch_length=(cell.obs[2]+cell.obs[3]))
+    else:
+        return Clade(branch_length=(cell.obs[2]+cell.obs[3]), clades=[fullRecursive(cell.left, a), fullRecursive(cell.right, a)])
```

Comment:
I know this is gonna be a little annoying, but will this function be able to handle `nans`? Consider a cell that started in G2, so it's G1 time observation is `nan` (`cell.obs[2]=float('nan')`). This will never show up in the synthetic lineages for the figures, but definitely shows up in the Heiser data. Something to consider for later.